game-core/src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -618,11 +618,10 @@ public void mouseEnter(final List<Unit> units, final Territory territory) {
           final boolean isCorrectTerritory =
               (firstSelectedTerritory == null) || firstSelectedTerritory.equals(territory);
           if (someOwned && isCorrectTerritory) {
-            final Collection<List<Unit>> highlight = new ArrayList<>();
-            highlight.add(units);
-            getMap().setUnitHighlight(highlight);
+            getMap()
+                .setUnitHighlight(Collections.unmodifiableList(Collections.singletonList(units)));
```

Comment:
I believe the order of those should be swapped

Reply:
updated: 0465ba106